easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1226,6 +1226,13 @@ def det_modpath_extensions(self, ec):
         self.log.debug("Obtained modulepath extensions: %s" % modpath_extensions)
         return modpath_extensions
 
+    def det_user_modpath_extensions(self, ec):
+        """Determine user-specific modulepath extensions according to module naming scheme."""
+        self.log.debug("Determining user modulepath extensions for %s" % ec)
+        modpath_extensions = self.mns.det_user_modpath_extensions(self.check_ec_type(ec))
+        self.log.debug("Obtained user modulepath extensions: %s" % modpath_extensions)
```

Comment:
use `self.log.debug("...", foo)` rather than `"..." % foo` (the former is slightly faster, string templating will only be done when the `log.debug` is actually 'performed')